examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb

```diff
@@ -154,7 +154,10 @@
     "# define the number of repeats for the hybrid solver to search for the optimal solution.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    num_reads = 1000 # default value set by D-wave is 1000 per task
</pre><p> </p><p>could we change the value to 100? Or does that break the example? </p><p> </p><pre class="ql-syntax" spellcheck="false">num_reads = 100 # The default value for D-Wave Advantage_system1 is 1000
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>